samples/highcharts/data/livedata-columns/demo.js

```diff
@@ -0,0 +1,19 @@
+
+Highcharts.chart('container', {
+    chart: {
+        type: 'spline',
+        height: 800
+    },
+    title: {
+        text: 'Live Data (Columns JSON)'
+    },
+
+    subtitle: {
+        text: 'Data input from a remote JSON file'
+    },
+
+    data: {
+        columnsURL: 'https://demo-live-data.highcharts.com/time-cols.json',
```

Comment:
The first item in the columns arrays should be the series name. Currently the data module interprets the first value as series name, so the series name constantly changes. I tried setting the `firstRowAsNames` option to `false`, but apparently this only applies to CSV.